staging/src/k8s.io/client-go/tools/clientcmd/api/types.go

```diff
@@ -74,6 +74,9 @@ type Cluster struct {
 	// CertificateAuthorityData contains PEM-encoded certificate authority certificates. Overrides CertificateAuthority
 	// +optional
 	CertificateAuthorityData []byte `json:"certificate-authority-data,omitempty"`
+	//HttpProxy holds the proxy server details for the cluster
```

Comment:
what is the exact format?

Reply:
It would be an` ip:port` similar to what the `Server` field in the Cluster currently stores.

Reply:
aren't proxies also http(s) servers? What about credentials?